sql_upgrade.php

```diff
@@ -69,101 +69,109 @@
 <html>
 <head>
 <title>OpenEMR Database Upgrade</title>
-<link rel='STYLESHEET' href='interface/themes/style_blue.css'>
-<link rel="shortcut icon" href="public/images/favicon.ico" />
+    <link rel="stylesheet" href="public/assets/bootstrap/dist/css/bootstrap.min.css">
+    <script src="public/assets/jquery/dist/jquery.min.js"></script>
+    <script src="public/assets/bootstrap/dist/js/bootstrap.bundle.min.js"></script>
+    <link rel="shortcut icon" href="public/images/favicon.ico" />
 </head>
 <body>
-<center>
-<span class='title'>OpenEMR Database Upgrade</span>
-<br />
-</center>
-<?php
-if (!empty($_POST['form_submit'])) {
-    $form_old_version = $_POST['form_old_version'];
+    <div class="container mt-3">
+        <div class="row">
+            <div class="col-12">
+                <h2>OpenEMR Database Upgrade</h2>
+            </div>
+        </div>
+        <div class="jumbotron p-4">
+            <form class="form-inline" method='post' action='sql_upgrade.php'>
+                <div class="form-group mb-3">
+                    <label>Please select the prior release you are converting from:</label>
+                    <select class='ml-3 form-control' name='form_old_version'>
+                        <?php
+                        foreach ($versions as $version => $filename) {
+                            echo " <option value='$version'";
+                            // Defaulting to most recent version, which is now 5.0.2.
+                            if ($version === '5.0.2') {
+                                echo " selected";
+                            }
+
+                            echo ">$version</option>\n";
+                        }
+                        ?>
+                    </select>
+                </div>
+                <p>If you are unsure or were using a development version between two
+                    releases, then choose the older of possible releases.</p>
+                <p class="text-danger">If you are upgrading from a version below 5.0.0 to version 5.0.0 or greater, do note that this upgrade can take anywhere from several minutes to several hours (you will only see a whitescreen until it is complete; do not stop the script before it is complete or you risk corrupting your data).</p>
+                <input type='submit' class='btn btn-primary' name='form_submit' value='Upgrade Database' />
```

Comment:
change to button and note since this script includes globals you have access to the fontawesome button icons :)

Reply:
Hi @bradymiller 

I include the `globals.php` file, change the input to button and add the `btn-update`,
but it seems odd, is correct?

![image](https://user-images.githubusercontent.com/22230889/83960636-7301bb00-a8bd-11ea-8536-c025c03dd322.png)

Reply:
Agreed. Maybe btn-transmit icon. If that looks funny also, and none of the others look right,, then wouldn't use one.

btw, the script already includes globals.php, so you shouldn't need to do that.